cpp/data/installer/control_number_guesser.sql

```diff
@@ -1,47 +0,0 @@
--- Control numbers are saved as a single text blob consisting of individual numbers concatenated together with the '|' character.
```

Comment:
Why has control_number_guesser.sql been removed?
(It is needed by lib/src/ControlNumberGuesser.cc)

Reply:
I'm sorry, I can't remember why I did that.  In any case it is back now.